core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1676,11 +1830,7 @@ public void onFastForward() {
         @Override
         public void onSkipToNext() {
             Log.d(TAG, "onSkipToNext()");
-            if (UserPreferences.shouldHardwareButtonSkip()) {
-                mediaPlayer.skip();
-            } else {
-                seekDelta(UserPreferences.getFastForwardSecs() * 1000);
-            }
+            skip();
```

Comment:
My understanding is that after PR #2504, the majority of these methods might not ever be called because the key that would trigger them is already handled via [handleKeycode](https://github.com/AntennaPod/AntennaPod/blob/c80c550214ac974f50976b961005a589460511c9/core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java#L1851).

This should be confirmed though. I can dig down into `MediaSessionCompat` some more.